### PR TITLE
haiiii:3

### PR DESCRIPTION
**Pull request by ItsEmpa**

### CODE REVIEW

File: src/main/java/at/hannibal2/skyhanni/features/misc/PetNametag.kt [hunk 1]
```diff
@@ -20,11 +20,12 @@ class PetNametag {
      */
     private val pattern by RepoPattern.pattern(
         "feature.pet.nametag",
-        "(?<start>§8\\[§7Lv(?<lvl>\\d+)§8\\]) (?<rarity>§.)(?<player>[\\w_]+'s?) (?<pet>[\\w\\s]+)(?<skin>§5 ✦)?"
+        "(?<start>§8\\[§7Lv(?<lvl>\\d+)§8]) (?<rarity>§.)(?<player>[\\w_]+'s?) (?<pet>[\\w\\s]+)(?<skin>§. ✦)?"
```
Comment (j10a1n15) [inline src/main/java/at/hannibal2/skyhanni/features/misc/PetNametag.kt:23]: why did you remove the \\\\

Comment (ItsEmpa) [reply src/main/java/at/hannibal2/skyhanni/features/misc/PetNametag.kt:23]: intellij was complaining

Comment (j10a1n15) [reply src/main/java/at/hannibal2/skyhanni/features/misc/PetNametag.kt:23]: okay but
its weird now

Comment (ItsEmpa) [reply src/main/java/at/hannibal2/skyhanni/features/misc/PetNametag.kt:23]: no u
